.github/ISSUE_TEMPLATE/1-bug.yml

```diff
@@ -21,7 +21,7 @@ body:
         - label: I have searched the existing issues
           required: true
 
-  - type: textarea
+  - type: input
```

Comment:
Have you actually tried running `mc -V`? I really want to have it all and as a fenced code block.

Reply:
yes, but i would have naturally used only the first line.
if you want all of it (which i guess is reasonable), then the description needs to mention "and build configuration".

Reply:
All right, feel free to suggest a better description. As I said, so far everyone has actually included the full output, so I was quite happy about that.